stdlib/lang-parser/README.md

```diff
@@ -0,0 +1,309 @@
+## Ballerina-lang-parser
+
+The parser is implemented following the LL(K) recursive Descent parsing pattern. This pattern analyses the syntactic structure using k>1 lookahead tokens.
```

Comment:
```suggestion
The parser is implemented following the LL(K) recursive descent parsing pattern. This pattern analyses the syntactic structure using k>1 lookahead tokens.
```

Reply:
Ballerina lang-parser implementation in no longer maintained and has already being removed from the ballerina-lang repo.